packages/mco/components/modals/protected-applications/managed-applications-modal.tsx

```diff
@@ -80,7 +81,9 @@ const ManagedApplicationsModal: React.FC<CommonModalProps> = (props) => {
           isInline
         >
           {t('Continue to Applications page')}
-          <ArrowRightIcon size={IconSize.sm} className="pf-u-ml-sm" />
+          <Icon size="sm">
+            <ArrowRightIcon className="pf-u-ml-sm" />
```

Comment:
there are other references in this file too...
```suggestion
            <ArrowRightIcon className="pf-v5-u-ml-sm" />
```